app/assets/stylesheets/pages/_home.scss

```diff
@@ -1 +1,17 @@
 // Specific CSS for your home-page
+section {
+  height: 667px;
```

Comment:
can we try to not use fixed px as it tends to mess up at other displays

Reply:
col-xs?